ui/lobby/src/hookRepo.ts

```diff
@@ -1,44 +1,45 @@
 import LobbyController from './ctrl';
 import { Hook } from './interfaces';
 
-function ratingOrder(a, b) {
+function ratingOrder(a: Hook, b: Hook) {
   return (a.rating || 0) > (b.rating || 0) ? -1 : 1;
 }
 
-function timeOrder(a, b) {
+function timeOrder(a: Hook, b: Hook) {
   return a.t < b.t ? -1 : 1;
 }
 
 export function sort(ctrl: LobbyController, hooks: Hook[]) {
   hooks.sort(ctrl.sort === 'time' ? timeOrder : ratingOrder);
 }
 
-export function init(hook: Hook) {
+export function init(hook: Hook): Hook {
```

Comment:
Currently reviewing on mobile. Should this just return void, if it didn't return anything before?

Reply:
Oops, that's a leftover I forgot to remove. I thought about changing this to return something of a new type since the incoming hooks don't actually have the action field set yet but decided that it's probably not worth it.